frontend/packages/core/src/AppLayout/header.tsx

```diff
@@ -1,46 +1,34 @@
 import React from "react";
 import { Link } from "react-router-dom";
-import {
-  AppBar as MuiAppBar,
-  Box,
-  Divider as MuiDivider,
-  IconButton,
-  Toolbar,
-  Typography,
-} from "@material-ui/core";
+import styled from "@emotion/styled";
+import { AppBar as MuiAppBar, Box, Grid, IconButton, Toolbar, Typography } from "@material-ui/core";
 import MenuIcon from "@material-ui/icons/Menu";
-import styled from "styled-components";
 
 import Drawer from "./drawer";
 import Logo from "./logo";
+import Notifications from "./notifications";
 import SearchField from "./search";
 import { UserInformation } from "./user";
 
-const AppBar = styled(MuiAppBar)`
-  ${({ theme }) => `
-  min-width: fit-content;
-  background-color: ${theme.palette.secondary.main};
-  color: ${theme.palette.primary.main};
-  min-width: fit-content;
-  `}
-`;
+const AppBar = styled(MuiAppBar)({
+  minWidth: "fit-content",
```

Comment:
You can leave a todo here since we don't design for mobile at the moment but on smaller devices the layout here doesn't stay consistent.
![Screen Shot 2020-12-02 at 8 19 52 PM](https://user-images.githubusercontent.com/1004789/100963680-c0ad5800-34db-11eb-873d-86101680440d.png)

Reply:
got it, will leave a to-do and follow up!